tests/modules/nf-core/glimpse/phase/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { GLIMPSE_PHASE } from '../../../../../modules/nf-core/glimpse/phase/main.nf'
+
+workflow test_glimpse_phase {
+    input_vcf = [
+        [ id:'input2', single_end:false ], // meta map
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/NA12878.chr21.s.1x.vcf.gz", checkIfExists: true),
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/NA12878.chr21.s.1x.vcf.gz.csi", checkIfExists: true),
+    ]
+    
+    ref_panel = [
+        [ id:'reference', single_end:false ], // meta map
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf", checkIfExists: true),
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/1000GP.chr21.noNA12878.s.bcf.csi", checkIfExists: true)
+    ]
+
+    GLIMPSE_PHASE (
+        input_vcf,
+        file("https://github.com/nf-core/test-datasets/raw/modules/data/delete_me/glimpse/chr21.b38.gmap.gz", checkIfExists: true),
+        [],
+        ref_panel,
+        [],
```

Comment:
Can you also add tests with these optional values filled in?

Reply:
I have to admit I don't really know how it works with those options. I will try to make some tests for it.

Reply:
Normally it should be done.
I've created 2 test, one without the option and the other with it.

Reply:
Yeah this is okay, Github just removed my suggestion and put this in its place :grin: I added the suggestion to my approve message :)